dojo/unittests/test_snyk_parser.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase
+from dojo.tools.snyk.parser import SnykParser
+from dojo.models import Test
+
+
+class TestSnykParser(TestCase):
+    def test_snykParser_without_file_has_no_finding(self):
+        parser = SnykParser(None, Test())
+        self.assertEqual(0, len(parser.items))
+
+    def test_snykParser_single_has_no_finding(self):
+        testfile = open("dojo/unittests/scans/snyk/single_project_no_vulns.json")
+        parser = SnykParser(testfile, Test())
+        self.assertEqual(0, len(parser.items))
+        testfile.close()
+
+    def test_snykParser_allprojects_has_no_finding(self):
+        testfile = open("dojo/unittests/scans/snyk/all-projects_no_vulns.json")
+        parser = SnykParser(testfile, Test())
+        self.assertEqual(0, len(parser.items))
+        testfile.close()
+
+    def test_snykParser_single_has_one_finding(self):
+        testfile = open("dojo/unittests/scans/snyk/single_project_one_vuln.json")
+        parser = SnykParser(testfile, Test())
+        self.assertEqual(1, len(parser.items))
+        testfile.close()
+
+    def test_snykParser_allprojects_has_one_finding(self):
+        testfile = open("dojo/unittests/scans/snyk/all-projects_one_vuln.json")
+        parser = SnykParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(1, len(parser.items))
+
+    def test_snykParser_single_has_many_findings(self):
+        testfile = open("dojo/unittests/scans/snyk/single_project_many_vulns.json")
+        parser = SnykParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(41, len(parser.items))
+
+    def test_snykParser_allprojects_has_many_findings(self):
+        testfile = open("dojo/unittests/scans/snyk/all-projects_many_vulns.json")
+        parser = SnykParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(4, len(parser.items))
+
+    def test_snykParser_finding_has_fields(self):
+        testfile = open("dojo/unittests/scans/snyk/single_project_one_vuln.json")
+        parser = SnykParser(testfile, Test())
+        testfile.close()
+        finding = parser.items[0]
+        self.assertEqual(
+            "com.test:myframework@1.0.0-SNAPSHOT: XML External Entity (XXE) Injection",
+            finding.title)
+        self.assertEqual(
+            "Medium", finding.severity)
+        self.assertEqual(
+            "Issue severity of: **Medium** from a base CVSS score of: **6.5**",
+            finding.severity_justification)
+        self.assertEqual(
+            "CVE-2019-12400",
+            finding.cve)
+        self.assertEqual(
+            "611",
```

Comment:
cwe attributes is an integer if you check the model
```suggestion
            611,
```